src/WebJobs.Script/Host/ScriptHost.cs

```diff
@@ -135,7 +135,7 @@ public string InstanceId
         /// Gets the collection of all valid Functions. For functions that are in error
         /// and were unable to load successfully, consult the <see cref="FunctionErrors"/> collection.
         /// </summary>
-        public virtual Collection<FunctionDescriptor> Functions { get; private set; }
+        public virtual Collection<FunctionDescriptor> Functions { get; private set; } = new Collection<FunctionDescriptor>();
```

Comment:
We've been seeing null refs from this property, and in my local testing I was also hitting them. We're accessing this property in several places w/o checking for null.